addons/mail/models/mail_thread.py

```diff
@@ -2444,7 +2443,9 @@ def _notify_prepare_template_context(self, message, msg_vals, model_description=
             if template and template.lang:
                 lang = template._render_lang([self.env.context['default_res_id']])[self.env.context['default_res_id']]
 
-        if not model_description and model:
+        if model_id:
```

Comment:
Not sure to understand this change ... I suppose the issue is that we propagate an already-translated model_description ?